### ISSUE TITLE
[FEAT] cloze overlapping

### ISSUE THREAD

**Issue by mkschulze** [st3v3nmw/obsidian-spaced-repetition#284]
**Is your feature request related to a problem? Please describe.**
When reviewing a multiple cloze card, it would be great to have the option to use a cloze overlapping, so all ==highlights== get hidden away.

**Describe the solution you'd like**
Having this card here:
```markdown
#cloze<br>
Condition 1: ==explanation for 1==<br>
Condition 2: ==explanation for 2==<br>
Condition 3: ==explanation for 3==<br>
Condition 4: ==explanation for 4==<br>
```
![Bildschirmfoto 2021-08-30 um 21 34 25](https://user-images.githubusercontent.com/3390820/131394827-c1d0d67f-c7cf-4501-8863-1b5a38596895.png)

It would be great to have all explanations hidden away.

**Additional context**
Feature as an alternative to a fixed chain repetition.

**Comment by mkschulze** [comment]
how's life coming along @st3v3nmw? :) 

I just aquired 1.100 flashcards, that want to get ported to obsidian-spaced-repitition and would be looking forward to using cloze overlapping. 😇🙏

**Comment by ucelote-de-mar** [comment]
This would be awesome, esp. if this can be done linewise like

> Nouns are words for **people**, **animals**, **places**, or **things**.

So I know beforehand I have to complete 4 items.

**Comment by mkschulze** [comment]
Still in favour :)

**Comment by mkschulze** [comment]
any news about this maybe?

**Comment by ronzulu** [comment]
Hi @MotaOcimar, I'm looking at all the issues you referenced in your PR https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/943 to gain an understanding as part of reviewing your great contribution.

I believe that the example within this issue would be satisfied with:
```
Condition 1: ==explanation for 1==[^1]<br>
Condition 2: ==explanation for 2==[^1]<br>
Condition 3: ==explanation for 3==[^1]<br>
Condition 4: ==explanation for 4==[^1]<br>
```

I.e. specifying the same sequence number `1` in each deletion. Is that correct?

Hi @mkschulze for clarity, I'm wondering why you use the term "overlapping" in your description. Perhaps I have missed something, but can't see anything overlapping.

**Comment by mkschulze** [comment]
@ronzulu I think what I was looking for is just having multiple ==HIGLIGHTS== hidden by the cloze instruction. So that it "overlaps" or covers all such instructions on the card. Maybe not the perfect word for it.

**Comment by ronzulu** [comment]
Hi @mkschulze thanks for the clarification.

**Comment by MotaOcimar** [comment]
Hi @ronzulu!

Thank you for taking a look at the issues related to Clozes!

Regarding the solution for this issue, I had the same understanding as you. Marking all clozes with the sequence number `1` seems satisfactory, and there would be no need to use the "overlapping" functionality.

Also, thank you @mkschulze for staying active here for clarifications!

Best,  
Ocimar

**Comment by st3v3nmw** [comment]
Hi,

I've just made a release [v1.13.0](https://github.com/st3v3nmw/obsidian-spaced-repetition/releases/tag/1.13.0) that ships with custom cloze patterns 🥳 all thanks to #943. The updated documentation is live [here](https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/cloze-cards/#cloze-types).

I've tested the release but not as extensively as I normally would. Perhaps we need a beta version of the plugin? That being said, I'm on standby to fix bugs ASAP. Just give me a shout.

Cheers.
